docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_Coverflow.htm

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
+	<link href="../styles.css" rel="StyleSheet" type="text/css">
+	<link href="../snippet.css" rel="StyleSheet" type="text/css">
+	<title>Coverflow</title>
+</head>
+<body onload="prettyPrint()" id="content">
+
+
+<h1>Coverflow</h1>
+
+<p>The coverflow component visually applies various effects to selected images.</p>
+
```

Comment:
To use coverflow, you must note that "tau.coverflow.js" should be imported with tau.js.
<script type="text/javascript" src="lib/tau/libs/tau.coverflow.js"></script>

Reply:
done.